extensions/escape-game/standalone.mjs

```diff
@@ -124,6 +146,33 @@ class Standalone {
     };
   }
 
+  async e2s_openDoor() {
+    var res = await this.helper.varUtils.writeVar("@doorLastOpen", null, null, Date.now().toString());
```

Comment:
You can use a member variable instead of varUtils, because the entire server shares 1 single instance of standalone server for escape-game.